More info on GetFeatureInfo WITH_GEOMETRY parameter (#5975)
qgis/QGIS-Documentation: docs/server_manual/getting_started.rst

text or GML format depends the output format chosen for the GetFeatureInfo request.

If you wish, you can check |checkbox| :guilabel:`Add geometry to feature response`.

> This will include bounding box for every feature in the GetFeatureInfo response.
> See :ref:`WITH_GEOMETRY <server_wms_getfeatureinfo>` parameter to receive geometries in text format.

Reviewer: ```suggestion
This will include the bounding box for each feature in the GetFeatureInfo response.
See also the :ref:`WITH_GEOMETRY <server_wms_getfeatureinfo>` parameter.
```
